### DISCUSSION TITLE
Why my GitHub account is blocked for signing up in Vercel?

### DISCUSSION THREAD

**Discussion by JunbinLi** [vercel/community#3319] [Help] [answered]
### Summary

I've never used Vercel before and when I was signing up my Vercel account, it says that my user account is blocked.

### Steps to Reproduce

goto vercel.com/signup
connect with my github account
the problem produces HERE!

**Comment by amyegan** [comment, marked as answer]
Hey there. Your login problem might be related to this incident: https://www.vercel-status.com/incidents/tp22kxv7rfcm

**Reply by JunbinLi** [reply]
But my email address connected with my GitHub account is also blocked without any connection with Vercel(except through GitHub)

**Reply by amyegan** [reply]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
